test/onedrive.test.js

```diff
@@ -18,8 +18,19 @@ const assert = require('assert');
 const OneDrive = require('../src/OneDrive.js');
 
 describe('OneDrive Tests', () => {
+  it('throws when required parameters are not specified.', async () => {
+    try {
+      // eslint-disable-next-line no-unused-vars
+      const drive = new OneDrive({});
+      assert.fail('This should not happen.');
+    } catch (e) {
+      // this is expected
```

Comment:
IIRC, `assert.fail()` will throw an exception. so the test passes in anycase. you need to check the error message or use [assert.throws()](https://nodejs.org/docs/latest-v10.x/api/assert.html#assert_assert_throws_fn_error_message)

Reply:
Thanks, done